### DISCUSSION TITLE
RDS not creating

### DISCUSSION THREAD

**Discussion by aambadkar** [learndevopsonline/learndevopsonline#1522] [Q&A] [answered]
hi,
i am trying to create an RDS but, it says parameter not found. in code file seems everything ok.
![Capture](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/2907a8f1-bf6c-4e2a-a1f3-6b9ee8fc6efe)
![Capture1](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/ee059d9b-a4ca-4940-8c09-a870a0a7df23)
![Capture2](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/6d44c8d1-9694-4c20-b20a-54d4ae8a185a)

**Comment by sreedharm07** [comment, marked as answer]
Hi @aambadkar 
you parameter name is with **dev.rds.master-password**. but you are calling parameter in code =**dev.rds.master_password** ( under score mistake) please change parameter name or your code with underscore

![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/a4cd6a0e-6d8d-4c74-b3c3-2fe2f7871b89)
